src/installer/tests/HostActivation.Tests/DependencyResolution/ResolveComponentDependencies.cs

```diff
@@ -49,6 +49,131 @@ public void ComponentWithNoDependenciesAndNoDeps()
                 .And.HaveStdErrContaining($"mgd_app='{component.AppDll}'");
         }
 
+        [Fact]
+        public void ComponentWithNoDependenciesCaseChangedOnAsm()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // Remove once https://github.com/dotnet/runtime/issues/42334 is resolved
```

Comment:
We have a fix for this in https://github.com/dotnet/runtime/pull/42879. Since it's test code it should be OK to also port that one over. It feels wrong adding a test which is semi-broken when we know how to fix it.

Reply:
Agreed, porting over test code is good.